CODE-OF-CONDUCT.md

```diff
@@ -37,7 +37,7 @@ projects.
 This code of conduct applies both within project spaces and in public spaces
 when an individual is representing the project or its community.
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a maintainer of the project, Dan Walsh <dwalsh@redhat.com>, and/or Tom Sweeney <tsweeney@redhat.com>.
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting a maintainer of the project, the Container Organization's [Code of Conduct Mailing List](codeofconduct@lists.podman.io), Dan Walsh <dwalsh@redhat.com>, and/or Tom Sweeney <tsweeney@redhat.com>.
```

Comment:
```suggestion
Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the Container Organization's [Code of Conduct Mailing List](codeofconduct@lists.podman.io).
```

I suggest to point to the list only (assuming Dan and you are on it in any case).  This avoids the potential bus factor when only one person is contacted and prevents the doc from out dating.

Reply:
I was on the fence with that.  I can see some cases where a person might feel comfortable reporting something to a person they knew, versus a list of 0 to n anonymous people.  How would they know if the person they were filing the complaint against wasn't on the list?  That was my main reason for keeping Dan and I on the list.

All that said, hopefully nothing anyone will ever need to use.  I'll try to remember to PS this at scrum.

Reply:
Tom and I avoid driving on the same bus. :^)

What do other sites do?